src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/TestingControllers/DataControllerTests.cs

```diff
@@ -322,10 +315,8 @@ private HttpClient GetTestClient()
             {
                 builder.ConfigureTestServices(services =>
                 {
-                    services.AddSingleton<IApplicationRepository, ApplicationRepositoryMock>();
-                    services.AddSingleton<IDataRepository, DataRepositoryMock>();
-                    services.AddSingleton<IInstanceEventRepository, InstanceEventRepositoryMock>();
-                    services.AddSingleton<IInstanceRepository, InstanceRepositoryMock>();
+                    services.AddMockRepositories();
```

Comment:
@acn-sbuad this is a place where it still makes sense perhaps? When the DI registrations are duplicated in multiple test classes

Reply:
Yeah, I agree.